language/German/strings.po

```diff
@@ -1167,37 +1150,37 @@ msgstr " (Standard)"
 #: src/emu/ui/selsoft.cpp:684
 #, c-format
 msgid "%1$s %2$s ( %3$d / %4$d softwares )"
-msgstr ""
+msgstr "%1$s %2$s ( %3$d / %4$d Softwares )"
```

Comment:
Einfach in den gleichen branch pushen aus dem du den Pull Request erstellt hast, die commits werden dann automatisch angehängt.